app/services/files.service.ts

```diff
@@ -49,7 +49,8 @@ function ensureNull(shouldBeNull: any): Promise<void> {
 export class LocalFileService implements FileService {
     getAppLocations(): Promise<AppLocations> {
         const currentDirectory = new LocalDirectory(app.getAppPath());
-        const pluginDirectory = new LocalDirectory(path.join(app.getPath("userData"), 'plugins'));
+        const pluginPath = process.env.ENV !== 'production' ? path.join('.', 'plugins') : path.join(app.getPath("userData"), 'plugins');
```

Comment:
This seems to work ok, at least for now. However, make sure to keep the code for copying over the directory since we will likely want to have a version of the app that comes with preinstalled plugins as suggested to me by @RighteousRaichu